std_msgs/Header header
franka_msgs/FrankaState franka_state
uint32 pt_idx
int16 is_valid
